scripts/check-repo-guard.ps1

```
@@ -0,0 +1,313 @@
+
+function Get-EventRange {
+    if ($Range) {
+        return $Range
+    }
+
+    $eventPath = $env:GITHUB_EVENT_PATH
+    if (-not $eventPath -or -not (Test-Path -LiteralPath $eventPath)) {
+        return $null
+    }
+
+    $event = Get-Content -LiteralPath $eventPath -Raw | ConvertFrom-Json
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)